BatteryMeter.c

```
@@ -25,34 +25,124 @@ static const int BatteryMeter_attributes[] = {
+      if (isCharging || isDischarging) {
+         ret = xSnprintf(
+            buf, len, "%.1fW @ %.1f/%.1fWh",
+            info.powerCurr, info.energyCurr, info.energyFull
+         );
+         buf += ret; len -= ret;
+
+         if (timeMinutes >= 0) {
+            ret = xSnprintf(
+               buf, len, ", %dh%02dm",
+               timeMinutes / 60, timeMinutes % 60
+            );
```

Cppcheck (2.20.0)

[style] The function 'isNaN' is never used.

(unusedFunction)